src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -23,4 +37,14 @@ private SearchFilterFiles() {}
 				.collect( Collectors.toList() ); //
 	}
 
+	public static List<Path> getProjectFilterFiles() throws IOException {
+		if ( !resolveFilterPath().toFile().exists() ) {
```

Comment:
You could extract `resolveFilterPath()` into a variable, then you would not need a method for this.